src/webln/NostrWeblnProvider.ts

```diff
@@ -213,6 +236,40 @@ export class NostrWebLNProvider {
       });
     });
   }
+
+  initNWC(provider: string, options: { name: string, returnTo?: string }) {
+    const height = 600;
+    const width = 400;
+    const top = window.outerHeight / 2 + window.screenY - height / 2;
+    const left = window.outerWidth / 2 + window.screenX - width / 2;
+    // @ts-ignore for my TS love :)
```

Comment:
you could make NWC_URLS be a `Record<string, string>` or make provider be a `keyof typeof NWC_URLS`. I think the latter is better because this will only work if a provider exists in NWC_URLs. Or, if you add another parameter to options so a user can provide a URL themselves?

Reply:
how do I do this? 

I wanted the NWC URLs to be preconfigured, but a custom URL should also be possible, true.

Reply:
It seems you already changed it and it's a bit more complicated now since it can be a URL or a provider name. I think the best you can easily do is:

```const urlStr = NWC_URLS[providerNameOrUrl as keyof typeof NWC_URLS] || providerNameOrUrl;```